config/routes.rb

```diff
@@ -199,18 +196,18 @@
   end
 
   # Help
-  get '/help/:group', :to => 'questions#index', :as => :help
-  get '/help' => redirect('/help/basics'), :as => :faq
+  get '/help/:group' => 'questions#index', as: :help
+  get '/help' => redirect('/help/basics'), as: :faq
 
   get '/metrics' => 'metrics#overview'
 
   # Guides
-  get '/guides/:group', :to => 'guides#index', :as => :guides
-  get '/guides', :to => 'guides#index'
+  get '/guides/:group' => 'guides#index', as: :guides
+  get '/guides' => 'guides#index'
 
   # redirect support-foo to helpful
-  get '/support-foo', to: redirect('/helpful')
-  get '/support-foo/*extra', to: redirect {|p, req| "/helpful/#{p[:extra]}" }
+  get '/support-foo' => redirect('/helpful')
+  get '/support-foo/*extra' => redirect {|p, req| "/helpful/#{p[:extra]}" }
```

Comment:
Unused block argument - `req`. If it's necessary, use `_` or `_req` as an argument name to indicate that it won't be used.<br>Space between { and | missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.